File: Objetos/objeto_jogador.js
//Objeto Jogador com os atributos nome, time, peças e jogadas
export class Jogador{
    constructor(id_jogador, nome, nomeUsuario, nascimento, genero, numero_partidas, vitorias, derrotas, partidas){
        this.id_jogador = id_jogador;
        this.nome = nome;
        this.nomeUsuario = nomeUsuario;
        this.nascimento = nascimento;
        this.genero = genero;
        this.numero_partidas = numero_partidas;
        this.vitorias = vitorias;
        this.derrotas = derrotas;
        this.partidas = partidas;
    }

    returnID_jogador(){
        return this.id_jogador;
    }

    returnNome(){
        return this.nome;
    }

    returnNomeUsuario(){
        return this.nomeUsuario;
    }

    returnNascimento(){
        return this.nascimento;
    }

    returnGenero(){
        return this.genero;
    }

    returnNumero_partidas(){
        return this.numero_partidas;
    }

    returnVitorias(){
        return this.vitorias;
    }

    returnDerrotas(){
        return this.derrotas;
    }

    returnPartidas(){
        return this.partidas;
    }
};

//Listas de IDs e jogadores
export let lista_ID_jogadores = [];
export let lista_jogadores = [];

File: Objetos/objeto_jogador.py
#Objeto Jogador com os atributos nome, time, peças e jogadas
class Jogador:
    def __init__(self, id_jogador, nome, nomeUsuario, nascimento, genero, numero_partidas, vitorias, derrotas, partidas):
        self.id_jogador = id_jogador
        self.nome = nome
        self.nomeUsuario = nomeUsuario
        self.nascimento = nascimento
        self.genero = genero
        self.numero_partidas = numero_partidas
        self.vitorias = vitorias
        self.derrotas = derrotas
        self.partidas = partidas
    

    def returnID_jogador(self):
        return self.id_jogador
    

    def returnNome(self):
        return self.nome
    

    def returnNomeUsuario(self):
        return self.nomeUsuario
    

    def returnNascimento(self):
        return self.nascimento
    

    def returnGenero(self):
        return self.genero
    

    def returnNumero_partidas(self):
        return self.numero_partidas
    

    def returnVitorias(self):
        return self.vitorias
    

    def returnDerrotas(self):
        return self.derrotas
    

    def returnPartidas(self):
        return self.partidas
    


#Listas de IDs e jogadores
lista_ID_jogadores = []
lista_jogadores = []